packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1676,3 +1741,113 @@ abstract class MultiSelectableSelectionContainerDelegate extends SelectionContai
     return finalResult!;
   }
 }
+
+/// A function that builds a widget to use as the text selection toolbar for
+/// [SelectableRegion].
+///
+/// See also:
+///
+///  * [ContextMenuBuilder], which is the generic type for any context menu
+///    builder, not just for the editable text selection toolbar.
+///  * [EditableTextToolbarBuilder], which is the builder for [EditableText].
+typedef SelectableRegionToolbarBuilder = Widget Function(
+  BuildContext,
+  SelectableRegionState,
+  Offset,
+  [Offset?]
+);
+
+/// Calls [builder] with the [ContextMenuButtonItem]s representing the
+/// buttons in this platform's default text selection menu.
+///
+/// By default the [targetPlatform] will be [defaultTargetPlatform].
```

Comment:
Since there's a build context available, shouldn't this use `Theme.of(context).platform`?

Reply:
This is the widgets library so I can't use Theme, right?